### ISSUE TITLE
Could not encrypt this message: Error: Error encrypting message!

### ISSUE THREAD

**Issue by ashnazg** [mailvelope/mailvelope#493]
I'm consistently receiving this error when trying to encrypt a message to one specific recipient (http://keys.gnupg.net/pks/lookup?op=get&search=0x14C85A0824EAF823).  This is the only public key that mailvelope is having trouble with for me.  Nothing is being recorded in the Options -> Log view.

**Comment by machine-mind** [comment]
I have the same Problem using one of my keys. As a side note: one of my keys recently expired and I renewed the expiration date key.

**Comment by toberndo** [comment]
@machine-mind This is a known limitation of OpenPGP.js. Workaround: you can do a `gpg --edit-key clean` on the key and re-import into Mailvelope. Make sure to delete the key first in Mailvelope.

**Comment by iiic** [comment]
Same problem for me :( but truly `clean`up helped :+1: 

deleting key from mailvelope, then editing the same key in gpg command line

```
gpg --import FILENAME
gpg --edit-key KEYID
clean
save
gpg --export --armor KEYID > FILENAME
```

and then import keyfile back into mailvelope

**Comment by toberndo** [comment]
Issue with multiple subkey binding signatures (e.g. when extending expiration date of key) is now fixed in Mailvelope v2.0.0
